### PR TITLE
Add Contributing Guide for New Contributors

### PR DESCRIPTION
**Pull request by vidhanarya**
In reference to Issue #155, This PR will add contribution guide and code of conduct to help newcomers who want to contribute get started. I have two new files i.e Code_of_Conduct.md and CONTRIBUTING.md files, and updated Readme.md with the link to contribution guidelines.
Note: Contributing guidelines are largely adapted from guidelines of [Atom Project](https://github.com/atom/atom/blob/master/CONTRIBUTING.md), [mlpack project](https://github.com/mlpack/mlpack/blob/master/CONTRIBUTING.md) and general Contributing guidelines [here](https://github.com/nayafia/contributing-template/blob/master/CONTRIBUTING-template.md)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+This project and everyone participating in it is governed by the [AerospaceResearch.net Code of Conduct](Code_of_Conduct.md). By participating, you are expected to uphold this code.
+
+## I don't want to read this whole thing I just have a question!!!
+
+> **Note:** Please don't file an issue to ask a question. You'll get faster results by using the resources below.
+
+We have an official [mailing list](https://lists.shackspace.de/mailman/listinfo/constellation) where you can post your question for the community.
+
+If chat is more your speed, you can join the Atom and Electron Slack team:
```
Comment (Nilesh4145) [inline CONTRIBUTING.md]: We don't have this I think.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+* **Describe the behavior you observed after following the steps** and point out what exactly is the problem with that behavior.
+* **Explain which behavior you expected to see instead and why.**
+* **Include screenshots and animated GIFs** which show you following the described steps and clearly demonstrate the problem.
+* **If the problem wasn't triggered by a specific action**, describe what you were doing before the problem happened.
+* **Include details about your configuration and environment.**
+
+
+### Suggesting Enhancements
+
+This section guides you through submitting an enhancement suggestion for Atom, including completely new features and minor improvements to existing functionality. Following these guidelines helps maintainers and the community understand your suggestion :pencil: and find related suggestions :mag_right:.
```
Comment (Nilesh4145) [inline CONTRIBUTING.md]: ~atom~ orbitdeterminator

File: CONTRIBUTING.md (hunk 1 above)
Comment (vidhanarya) [reply CONTRIBUTING.md]: Should I remove Constellation mailing list part??

**Comment by vidhanarya** [comment]
I've resolved all the changes, why is it still showing changes requested, is there anything that I missed??

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
+# Contributing to Atom
```
Comment (Nilesh4145) [inline CONTRIBUTING.md]: ~Atom~ Orbitdeterminator

Comment (vidhanarya) [reply CONTRIBUTING.md]: It was resolved in commit [#89368d9](https://github.com/aerospaceresearch/orbitdeterminator/pull/159/commits/89368d9fcc56291b922068a9e832048871a77b07)

**Pull request merged**